### PR TITLE
shortTheme and summaryItem styles modified, fixed showed themes on ku…

### PR DESCRIPTION
**Pull request by ruut975**
…ntavaalit page

### CODE REVIEW

**Comment by ruut975** [comment]
#### 📝Changed routes:


| [**/teemat/\[\[...uid\]\]**](https://paula-himanen-hyqsdjkbw-ruut975.vercel.app/teemat/[[...uid]]) |
|:-:|
|<a href="https://paula-himanen-hyqsdjkbw-ruut975.vercel.app/teemat/[[...uid]]">Screenshot of /teemat/[[...uid]]</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/_wOjvU8EEMtT24aXwqMTNGXyVsQ_.png">(view full size)</a> |







Commit 7a57ae2d919ffd6acdf9177870730dd8487b129c (https://paula-himanen-hyqsdjkbw-ruut975.vercel.app).

File: components/Tabs/TabBody/TabBody.js [hunk 1]
```diff
@@ -9,7 +9,9 @@ const TabBody = ({ currentTab }) => {
       return (
         <ThemeShort
           key={item.id}
-          theme={item}
+          tab_text={item.tab_text}
```
Comment (giograf) [inline components/Tabs/TabBody/TabBody.js:12]: tabText is proper notation. Make sure it is camelCase

File: components/slices/SummaryItems/SummaryItem/SummaryItem.js [hunk 2]
```diff
@@ -14,9 +14,11 @@ const SummaryItem = ({
 }) => {
   return (
     <div className={styles.summaryItem}>
-      <h1 className={styles.summaryTitle}>{title}</h1>
-      <div className={styles.imageWrapper}>
-        <Image src={url} alt={alt} width={width} height={height} />
+      <div className={styles.gridContainer}>
```
Comment (giograf) [inline components/slices/SummaryItems/SummaryItem/SummaryItem.js:17]: Before you had a wrapper for inner elements already. I believe that the introduction of new wrapper - gridContainer may be unnecessary. Check if you can get away with just having the parent container.

Comment (ruut975) [reply components/slices/SummaryItems/SummaryItem/SummaryItem.js:17]: The grid container is needed to center the title in the middle of the image. and the outer wrapper is flex container and needed to place the image and texts side by side.

**Pull request merged**